src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "11.3.0-prerelease",
+  "version": "11.4.0-prerelease",
```

Comment:
After a second thought, I think this is OK, as it's ```prerelease```, so we don't need to update its major version.